Makefile

```diff
@@ -15,7 +15,7 @@
 EXTENSION_PREFIX            := gardener-extension
 NAME                        := networking-cilium
 ADMISSION_NAME              := admission-cilium
-REGISTRY                    := eu.gcr.io/gardener-project/gardener
+REGISTRY                    := europe-docker.pkg.dev/gardener-project/releases
```

Comment:
Why does this differ from how we do it for `calico`, i.e. why not use `europe-docker.pkg.dev/gardener-project/public/gardener`?

Reply:
changed to `/public/gardener`, as suggested. btw: both previously, and after this change, `registry` is not an adequate term according to naming according to OCI.